automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataDownloadSpec.scala

```diff
@@ -154,42 +138,53 @@ class DataDownloadSpec extends FreeSpec with ParallelTestExecution with WebBrows
                                    userHidden: Option[String] = None,
                                    importColumns: Option[List[String]] = None,
                                    expectedColumns: List[String],
-                                   downloadPath: String)
-                                  (implicit webDriver: WebDriver): Unit = {
+                                   downloadPath: String): Unit = {
+
     val owner = UserPool.chooseProjectOwner
     val writer = UserPool.chooseStudent
     implicit val authToken: AuthToken = owner.makeAuthToken()
 
     withCleanBillingProject(owner) { billingProject =>
       withWorkspace(billingProject, "DataSpec_download", aclEntries = List(AclEntry(writer.email, WorkspaceAccessLevel.Writer))) { workspaceName =>
         api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
-        withSignIn(owner) { _ =>
-          (defaultShown, defaultHidden) match {
-            case (None, None) => Unit
-            case (_, _) => setColumnDefaults(billingProject, workspaceName, defaultShown.getOrElse(List()), defaultHidden.getOrElse(List()))
-          }
 
-          val dataTab = new WorkspaceDataPage(billingProject, workspaceName).open
-          dataTab.importFile(generateMetadataFile(initialColumns))
-          userHidden.foreach(dataTab.dataTable.hideColumn)
-        }
+        withWebDriver(downloadPath) { implicit driver =>
```

Comment:
use `withWebDriver` before UI login. `waitForBucketReadAccess` could take too long of time then causes hang error from browser time out.